usecases/encryption-at-rest.markdown

```diff
@@ -0,0 +1,26 @@
+# Encryption At Rest
+
+## Why
+
+Apache Kafka does not directly support any form of encryption for data stored within a broker. This means that the contents
+of records sent to Apache Kafka are stored in the clear. Anyone with sufficient priviledges, such as a Kafka Adminsitrator,
+is able to read the contents of the record.  If the information within the records is Personally Identifiable Information (PII)
+or otherwise confidential, this presents a problem.
+
```

Comment:
mention something about Governmental regulations such as GDPR, and industry standards HIPAA and PCI/DSS